pkg/build/controller/controller.go

```diff
@@ -220,7 +223,7 @@ func (bc *BuildPodController) CancelBuild(build *buildapi.Build, pod *kapi.Pod)
 		return err
 	}
 
-	glog.V(2).Infof("Build %s was successfully cancelled.", build.Name)
+	glog.V(2).Infof("Build %s/%s was successfully cancelled.", build.Namespace, build.Name)
```

Comment:
without getting into the specifics of this case, my view is that v(4) is going to be so verbose that it's the "hail mary" log mode (anything that's printed as part of every event processing or resync flow should be level 4).  for things that are rarer (error flows, infrequent events) it would be good to use a higher level (eg 2 or 3) so we can provide an administrator with a log level that they can leave turned on full time without flooding their logs but still catching critical things if we're trying to chase an issue.

(admittedly it might be better if what i described as v(4) be logged at v(5) instead).